[2024-09-10T07:11:05.986+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T07:11:06.003+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-11-18T00:00:00+00:00 [queued]>
[2024-09-10T07:11:06.019+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-11-18T00:00:00+00:00 [queued]>
[2024-09-10T07:11:06.020+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T07:11:06.037+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): first_task> on 2021-11-18 00:00:00+00:00
[2024-09-10T07:11:06.046+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1912) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T07:11:06.047+0000] {standard_task_runner.py:72} INFO - Started process 1919 to run task
[2024-09-10T07:11:06.046+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2021-11-18T00:00:00+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpowclrz94']
[2024-09-10T07:11:06.048+0000] {standard_task_runner.py:105} INFO - Job 116: Subtask first_task
[2024-09-10T07:11:06.100+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2021-11-18T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T07:11:06.204+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-18T00:00:00+00:00'
[2024-09-10T07:11:06.205+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T07:11:06.232+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T07:11:06.233+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-10T07:11:06.249+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T07:11:06.254+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-10T07:11:06.255+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T07:11:06.295+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T07:11:06.296+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2021-11-18T00:00:00+00:00, execution_date=20211118T000000, start_date=20240910T071106, end_date=20240910T071106
[2024-09-10T07:11:06.383+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T07:11:06.400+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T07:11:06.404+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
